frontend/packages/dev-console/src/components/pipelines/PipelineRuns.tsx

```diff
@@ -6,17 +6,18 @@ import {
   pipelineRunFilterReducer,
   pipelineRunStatusFilter,
 } from '../../utils/pipeline-filter-reducer';
+import { ListFilterIds, ListFilterStatuses } from '../../utils/pipeline-utils';
 import { PipelineRunModel } from '../../models';
 
-const filters = [
+export const RunFilters = [
```

Comment:
`RunFilters` could probably be named `runFilters` as it's not a class/component.